clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java

```diff
@@ -37,8 +37,18 @@
 abstract class BaseSchemaCatalog extends CatalogDTO implements Catalog, SupportsSchemas {
   /** The REST client to send the requests. */
   protected final RESTClient restClient;
+
+  /**
+   * Get the namespace of the current catalog.
+   *
+   * @return The namespace of the current catalog.
+   */
+  public Namespace namespace() {
+    return namespace;
+  }
```

Comment:
Don't add the method here. You'd better group fields and methods separately, not mixed them together.

Reply:
Also, why do we need catalog namespace and what is the catalog namespace? You also have a `schemaNamespace` variable below, is seems duplicated.

Reply:
> Don't add the method here. You'd better group fields and methods separately, not mixed them together.

ok, this might be generated by IDE there;  I will move it to bottom part, together with other methods.

Reply:
I removed the parameter "schemaNamespace" as its name may introduce confusing.